scripts/just.config.ts

```diff
@@ -44,6 +44,20 @@ function basicPreset() {
   option('push', { default: true });
 }
 
+/** Resolve whereas a storybook config + stories exist for a given path */
+function checkForStorybookExistence() {
+  const dir = process.cwd().replace(/\\/g, '/');
+  const packageName = dir.substring(dir.lastIndexOf('/') + 1);
+
+  // Returns true if the current package has a storybook config or the examples package has a storybook config and
+  // contains a folder with the current package's name.
+  return (
+    !!resolveCwd('./.storybook/main.js') ||
+    (!!resolveCwd('../examples/.storybook/main.js') &&
+      fs.existsSync(path.join(process.cwd(), `../examples/${packageName}`)))
```

Comment:
Probably better to use the path from the repo root (`findGitRoot()`) here to be safe